src/utils/permissions.ts

```diff
@@ -1,19 +1,19 @@
 import { CommandInteraction, PermissionString } from 'discord.js';
 
-export function hasPermissions(
-	permissions: PermissionString[],
+export function getMissingPermissions(
+	permissions: PermissionString[] | undefined,
```

Comment:
```suggestion
	permissions: PermissionString[] | null,
```

So, I think it's better to return null here.